tracing-attributes/tests/instrument.rs

```diff
@@ -238,3 +239,38 @@ fn impl_trait_return_type() {
 
     handle.assert_finished();
 }
+
+#[test]
+fn target_name_ident() {
```

Comment:
Could we split this into 2 or 3 tests please? One for `name`, another for `target` and if you believe it's necessary then a 3rd one for both (I'll leave that up to you).

Reply:
Done.